docs/pages/eas-insights/introduction.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: EAS Insights
+sidebar_title: Introduction
+hideTOC: true
+description: An introduction to EAS Insights which is a preview service for projects using the expo-insights library.
+---
+
+**EAS Insights** is a service that offers a view into a project's performance,
+usage, and reach. We are currently offering a preview of Insights that is
+available to all developers, and we will continue to roll out new features and
+functionality based on user feedback and suggestions.
+
+EAS Insights makes it easy to see the state of your app, providing information
+about usage across platforms, app store versions, and timeframes.
+
+### Integration with EAS Update
+
+For those using `expo-updates` with EAS Update, we're able to provide certain
+high-level usage insights without any additional client-side changes. This is
+possible due to the nature of EAS Update + expo-updates, where clients check for
+available updates and fetch them if possible. Data about these client requests
+is aggregated into a limited Insights view that shows usage over time, as well
+as usage broken down by platform.
```

Comment:
```suggestion
possible because by aggregating data from client requests to check for an update into a limited Insights view that shows usage over time, as well
as usage broken down by platform.
```

Reply:
(you can see here one of the downsides of manually wrapping docs)